### PR TITLE
Feature3

### PR DESCRIPTION
**Pull request by kenkuo86**
New features, coool

### CODE REVIEW

**Comment by jimc1200** [comment]
this is wrong

**Comment by pohua0606** [comment]
wow so amazing

**Comment by jerry861200** [comment]
It's not cool

**Comment by pohua0606** [comment]
Agree

**Comment by jimc1200** [comment]
agreed

File: hello.html [hunk 1]
```diff
@@ -6,5 +6,6 @@
 </head>
 <body>
 	<p>What's up nigga</p>
+	<h1>Feature3, edited by Ken</h1>
```
Comment (jimc1200) [inline hello.html:9]: what is this

Comment (pohua0606) [reply hello.html:9]: shut the fuck up

**Pull request merged**